PyAlbany/src/Albany_Pybind11_Timer.hpp

```diff
@@ -0,0 +1,22 @@
+//*****************************************************************//
+//    Albany 3.0:  Copyright 2016 Sandia Corporation               //
+//    This Software is released under the BSD license detailed     //
+//    in the file "license.txt" in the top-level Albany directory  //
+//*****************************************************************//
+
+#ifndef PYALBANY_TIMER_H
+#define PYALBANY_TIMER_H
+
+#include "Teuchos_StackedTimer.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+using RCP_StackedTimer = Teuchos::RCP<Teuchos::StackedTimer>;
+using RCP_Time = Teuchos::RCP<Teuchos::Time>;
+
+RCP_Time createRCPTime(const std::string name);
```

Comment:
can you please add descriptions of the functions pointing out any nonintuitive behavior?